charts/console/README.md

```diff
@@ -0,0 +1,13 @@
+# Redpanda Console Helm Chart
+
+This Helm chart allows you to deploy Redpanda Console to your Redpanda cluster.
+You can install the chart by running the following commands:
+
+```shell
+helm repo add redpanda 'https://charts.redpanda.com/' 
+helm repo update
+helm install redpanda/console -f myvalues.yaml
+```
+
+Have a look at the [values.yaml](./console/values.yaml) file to see the available options.
+Additionally, there is an example configuration in the [examples](./examples) directory.
```

Comment:
My vote would be to remove the `examples` folder since the only actual example there at the moment is around external access (and needs improvement and is outdated given https://github.com/redpanda-data/helm-charts/pull/220). I've been talking with @JakeSCahill and there are updates to our Kubernetes docs incoming that would replace https://github.com/redpanda-data/helm-charts/blob/main/examples/with-external-via-nodeport/README.md (one such update is https://github.com/redpanda-data/documentation/pull/897).

Reply:
Yes we will, but I'm not making any material changes to the chart in this migration PR. Reorganization can come after.